.github/workflows/closed_issue_check.yml

```diff
@@ -0,0 +1,51 @@
+# Contains job to check that all the todos are resolved for the closed issue.
+
+name: Closed Issue Check
```

Comment:
I suggest instead framing the name, documentation, and file to be based around issue workflows, instead. One specific example is for closed issues, but we may add more in the future (generic buckets for actions workflows are preferred over more specific ones).

Reply:
Done.